guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -34,6 +34,13 @@ Clients will remain supported.
 For more details, see the https://community.theforeman.org/t/drop-debian-11-ruby-2-7-and-nodejs-14-support-in-foreman-3-14/40503[removal RFC].
 endif::[]
 
+=== {awx} Parameter Change
+The `ansible_tower_fqdn` parameter has been removed and replaced with `ansible_tower_api_url`.
+This new parameter includes both the previous `ansible_tower_fqdn` value and the required API path for {awx}.
+
+By default, the API path is set to `/api/controller/v2` as used by {awx}{nbsp}2.5 and newer.
```

Comment:
This now expands to `AWX` in upstream (and is unused in downstream). Given the different versioning I now think it'd be best to expand it manually
```suggestion
By default, the API path is set to `/api/controller/v2` as used by Ansible Automation Controller{nbsp}2.5 and newer.
```

Reply:
But AAC and AAP are two different things. See https://access.redhat.com/support/policy/updates/ansible-automation-platform#packages-and-versions.